Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-A_user_executed_multiple_LDAP_enumeration_queries_Test.yml

```diff
@@ -890,8 +885,7 @@ tasks:
     skipunavailable: false
     task:
       brand: ''
-      description: Saves a context key indicating the user does not belong to default
-        privileged AD groups.
+      description: Saves a context key indicating the user does not belong to default privileged AD groups.
```

Comment:
Saves a context key indicating the user does not belong to default privileged Active Directory groups.